src/styles.css

```diff
@@ -360,6 +357,16 @@
   margin-bottom: 0;
 }
 
+/* Select Instructions */
+.cio-select-question-instructions {
+  text-align: center;
+  font-size: 1.125rem;
+  color: var(--gray-dust-100);
+  line-height: 150%;
+  margin-top: 2.75rem;
```

Comment:
I couldn't find the space pixels exactly from the designs but just by looking the margin-top seems a bit too much